client/common/sass/components/_buttons.sass

```diff
@@ -125,10 +125,13 @@
 		font-weight: 400
 		line-height: 1.23
 		padding: 0.25rem 0.5rem
-		background-color: $accent-background-tags
+		background-color: $tag-inactive-bg
```

Comment:
Hmmm, I see Accurat also was inconsistent about this, seemingly deliberately. Possibly the idea here was to make the database page less distracting? Or possibly just a quirk of the design.

![image](https://github.com/freedomofpress/pressfreedomtracker.us/assets/296631/b13f79ee-895f-4bda-a607-c125a487057b)

I do like the idea of increasing the contrast which I think will help with the disabled vibe, so let's do that. But lets leave it inconsistent for now (yellow on the homepage, gray on the database) and see how it is to live with it.

Reply:
I would probably just recommend using `$black` for text instead of finding a shade of gray. Even with an AA contrast level color (e.g. #6b6b6b), it looks pretty inactive to me.

Reply:
And in database page, yellow is also used as the color for selected filters, so could be that was the reason for not using yellow background for tag?

Reply:
Personally I am in favor of keeping it yellow the way it originally was for the whole site, because I do agree the gray makes it feel kind of like a disabled button. But curious to hear what you think @harrislapiroff